moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -158,6 +159,12 @@ const ProjectWorkActivitiesTable = () => {
   const [editActivity, setEditActivity] = useState(null);
   const { projectId } = useParams();
 
+
+  // new code /////
```

Comment:
Could you remove this comment and the ones on lines 166, 192, and 209? You might have planned to do this already before merge but I'd like to limit comments to ones that help future developers understand the intent of the code. I think that your naming of the functions and state variables does a great job of that on their own. 💯